src/ui/Menus/PostponeMenu.ts

```diff
@@ -38,11 +38,23 @@ export class PostponeMenu extends TaskEditingMenu {
             itemNamingFunction: NamingFunction,
             postponingFunction: PostponingFunction,
         ) => {
+            let postponeDateEqualsTaskFieldDate = false;
+            const dateFieldToPostpone = getDateFieldToPostpone(task);
```

Comment:
@claremacrae I'm aware that this code is also called in `public static PostponeMenu.postponeOnClickCallback()`, but I'd rather refactor this together afterwards since you know this code much better.

Reply:
Ok, I may try that, but definitely not in the PR, let's keep it small. Added a TODO =)